jenkins/test-image/README.md

```diff
@@ -0,0 +1,18 @@
+# TestImages
+
+From Kubernetes 1.4, we are making one test-images per Kubernetes version.
+
+Naming:
+User can add more Dockerfile and name it to Dockerfile-[TAG].
+[TAG] will be used as part of the built image tag.
+
+User can build one more all images, for more details run make help, or see "Usage" below.
+
+# Usage
+make build-all:      builds all the Dockerfile-*
+
+make push-all:       builds all the Dockerfile-* and pushes to k8s-testimages
+
+make build-single [Dockerfile-sample]:   builds Dockerfile-sample
```

Comment:
do you give the full Dockerfile filename? just giving the tag would probably be nicer.